### DISCUSSION TITLE
Good but...

### DISCUSSION THREAD

**Discussion by alcopsy** [Special-Niewbie/HandleOS#2] [General]
BCD function broken on dualboot (I had kept my AtlasOs but it blocks before choosing the windows to start if we activate the BCD functions, I have to reinstall everything).

Network optimization function button remains off after reboot, So I don't know if it was activated. 

I couldn't find the secret option of AMD adrenaline after enabling it 

Win+V is disable.. but needed a lot

Please add some langage (French) It's a pain to have to install language packs and set regions and time. 
I'll say more when I reinstall :)

**Comment by Special-Niewbie** [comment]
Thank you to try @alcopsy feels sorry to hear that. The BCD, it's aggressive modification must to be use with single Windows OS installation (I will write on the section). Before you reinstall everything, please try to follow the fix page about this problem. The Adrenaline if you try on HandleOS, if you are sure about that we need to check if your address that turn on this option after reboot it's changed the address. If the key it's in another address let's investigate together, where come from this. If you need more fast responses I opened the Discord Server, and let's go through together...

https://discord.com/invite/86N6WVnS

The network it's applied only if you keep your device in the  same IP network otherwise will go back to off. I made this to don't be too invasive

**Reply by alcopsy** [reply]
Thanks for the invitation! The fixed BCD worked perfectly :) however I have a fixed IP that does not change thanks to the MAC address added to my modem.This discord should be shared everywhere... I hadn't found it.

**Reply by Special-Niewbie** [reply]
The Discord Server it's just opened, for that reason I didn't even wrote on the main page :). But soon will come out a new version of it, anf I will mention about the Discord Server as well. Happy to hear that you have fixed everything. I hope to see you in the community.
